resources/views/layouts/app.blade.php

```diff
@@ -129,5 +129,6 @@
 <script src="{{ asset('js/jquery/jquery.min.js') }}"></script>
 <script src="{{ asset('js/tether/tether.min.js') }}"></script>
 <script src="{{ asset('js/bootstrap/bootstrap.min.js') }}"></script>
+<script id="dsq-count-scr" src="//classifiedz-xyz.disqus.com/count.js" async></script>
```

Comment:
what does the async do?

Reply:
I have no clue, but this line of code was given by disqus.